admin_manual/installation/source_installation.rst

```diff
@@ -47,7 +47,7 @@ Required
 PHP Version
 ~~~~~~~~~~~
 
-PHP >= 5.6 (ideally 7.0 or above)
+PHP (5.6+, 7.0, & 7.1)
```

Comment:
From @phil-davis 
> So words/numbers being re-formatted here would be best to reflect that PHP 7.1 is "encouraged/recommended" and that the older version support is just to give people time to change.

I highly propose reflecting the comment of @phil-davis like:
`` ownCloud recommends the use of PHP 7.1 in new installations. Sites using a version lower than PHP 7.1 are being strongly encouraged" to migrate to PHP 7.1``

Reply:
@mmattel, I've added the note that you recommended, based on @phil-davis suggestion.